lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -369,7 +369,9 @@ def plot_msd(self, max_lag=None, **kwargs):
         plt.xlabel("Lag time [s]")
         plt.ylabel(f"Mean Squared Displacement [{self._kymo._calibration.unit_label}$^2$]")
 
-    def estimate_diffusion(self, method, max_lag=None):
+    def estimate_diffusion(
+        self, method, max_lag=None, localization_variance=None, localization_variance_variance=None
```

Comment:
Naming (see above).

Reply:
`variance_of_localization_variance` it is

Reply:
👍